src/gsy_e/constants.py

```diff
@@ -22,12 +22,13 @@
 import os
 
 from gsy_framework.constants_limits import DATE_TIME_FORMAT, DATE_TIME_UI_FORMAT, TIME_ZONE  # NOQA
-from gsy_framework.constants_limits import TIME_FORMAT, DATE_FORMAT, GlobalConfig  # NOQA
+from gsy_framework.constants_limits import (  # NOQA
+    TIME_FORMAT,
+    DATE_FORMAT,
```

Comment:
Are these 2 and the NOQA required?

Reply:
I had to check myself again, too.
yes, we have a lot of imports in other modules that source these ones:
```python
from gsy_e.constants import TIME_ZONE
```
I am not sure whether this is good practice. Should I remove them and directly import from gsy-framework?

Reply:
I think that the best practice is to remove them from here actually and directly import them. I checked too and I think that the only reason why we had them like these were in order to use them as `gsy_e.constants.TIME_FORMAT`, however unless I do not see something, this can be replaced by a direct import. Therefore I would reecommend to take the time and remove them, and thanks!

Reply:
ok, will do.